spec/requests/set_user_after_login_spec.rb

```diff
@@ -11,7 +11,10 @@
       get '/auth/myusa/callback'
     }.to change { User.count }.by(1)
 
-    expect(User.last.email_address).to eq('george-test@example.com')
+    new_user = User.last
+    expect(new_user.email_address).to eq('george-test@example.com')
+    expect(new_user.first_name).to eq("Georgie")
```

Comment:
should we create a fixture without first name and last name to make sure nothing errors out when they are not present?